### ISSUE TITLE
[Bug?]: Vertical tabs a11y failures

### ISSUE THREAD

**Issue by caseybaggz** [pluralsight/design-system#2042]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

Vertical tabs are failing a11y checks for Verizon.

### Steps to reproduce 🕹

Steps:

1.
2.
3.
4.

**Comment by caseybaggz** [comment]
All issues currently fixed in latest version.
